### DISCUSSION TITLE
Github mobile

### DISCUSSION THREAD

**Discussion by idmnzr** [community/community#120011] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.158.0 (128225528)
Device: iPhone 8 Plus (iOS 16.7.7)
</details>

**Comment by ebndev** [comment, marked as answer]
Hi @idmnzr, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
